niri-config/src/lib.rs

```diff
@@ -2953,6 +2976,10 @@ mod tests {
             cursor {
                 xcursor-theme "breeze_cursors"
                 xcursor-size 16
+                hide-on-key-press
+                inactive-timeout {
+                    duration-ms 3000
+                }
```

Comment:
I think a better way is to replace this block with something like `hide-after-inactive-ms 3000`

Reply:
Should the inactive-timeout block of the previous layer be removed?

What I mean is that if a user needs to disable this feature, they should:

```plain
cursor {
  duration-ms -1 // Oh, I hate Magic Numbers!
}
```

or

```plain
cursor {
  inactive-timeout {
    off
    // hide-after-inactive-ms 3000
  }
}
```

Reply:
To disable the timeout, just don't put it in the config. This is consistent with other things like `focus-follows-mouse` and so on. In the struct, it should be an Option, then None would mean it's disabled and Some(u32) means it's enabled with that timeout.

Reply:
Oh I know what you mean. We can use `Options<T>`. My bad.